src/ui/components/Overlay/styles.scss

```diff
@@ -1,5 +1,7 @@
 @import '~photon-colors/photon-colors';
 
+$z-index: 1000;
```

Comment:
We should probably create a mixin someday to [manage z-index across the whole site](https://www.smashingmagazine.com/2014/06/sassy-z-index-management-for-complex-layouts/). It's not a pressing problem yet.